src/ClusterBootstrap/template/kubelet/kubelet.service.template

```diff
@@ -22,17 +22,14 @@ ExecStartPre=/bin/bash -c 'if  lspci | grep -qE "[0-9a-fA-F][0-9a-fA-F]:[0-9a-fA
 # 
 # https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/
 ExecStart=/opt/bin/kubelet \
-  --require-kubeconfig=true \
   --container-runtime={{'remote' if cnf["kube_custom_cri"] else 'docker'}} \
   --enable-server=true \
   --register-node=true \
-  --feature-gates="Accelerators=true,DevicePlugins=true" \
-  --allow-privileged=true \
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.11.md#before-upgrading

> The Kubelet's deprecated --allow-privileged flag now defaults to true. This enables users to stop setting --allow-privileged in order to transition to PodSecurityPolicy. Previously, users had to continue setting --allow-privileged, because the default was false.